pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/errorprone/AvoidFieldNameMatchingMethodNameRule.java

```diff
@@ -28,29 +31,36 @@ public Object visit(ASTClassOrInterfaceDeclaration node, Object data) {
     }
 
     @Override
-    public Object visit(ASTClassOrInterfaceBody node, Object data) {
-        int n = node.getNumChildren();
-        List<ASTFieldDeclaration> fields = new ArrayList<>();
-        Set<String> methodNames = new HashSet<>();
-        for (int i = 0; i < n; i++) {
-            Node child = node.getChild(i);
-            if (child.getNumChildren() == 0) {
-                continue;
-            }
-            child = child.getChild(child.getNumChildren() - 1);
-            if (child instanceof ASTFieldDeclaration) {
-                fields.add((ASTFieldDeclaration) child);
-            } else if (child instanceof ASTMethodDeclaration) {
-                methodNames.add(((ASTMethodDeclaration) child).getName().toLowerCase(Locale.ROOT));
+    public Object visit(ASTEnumDeclaration node, Object data) {
+        return super.visit(node, data);
```

Comment:
This method now merely calls super, so we don't need to override it

Reply:
I see. I'll remove this method. 😊